src/main/java/saps/common/utils/ExecutionScriptTag.java

```diff
@@ -10,12 +10,17 @@
 	private String name;
 	private String dockerRepository;
 	private String dockerTag;
+	private String memoryUsage;
+	private String cpuUsage;
 
-	public ExecutionScriptTag(String name, String dockerRepository, String dockerTag, String type) {
+	public ExecutionScriptTag(String name, String dockerRepository, String dockerTag, String memoryUsage,
+			String cpuUsage, String type) {
 		this.name = name;
 		this.dockerRepository = dockerRepository;
 		this.dockerTag = dockerTag;
 		this.type = type;
+		this.memoryUsage = memoryUsage;
```

Comment:
I'm missing the arg checks on this constructor (i'm afraid we have this problem spread on the whole codebase). As these parameters have some sort of defined pattern (16g, g means gigabytes and so on) we need to check the format.

In addition to that, Usage does not seem a good name. Is it a requirement, correct?

Reply:
Correct, I made some changes, review again